core/services/vrf/integration_v2_test.go

```diff
@@ -675,6 +658,26 @@ func TestVRFV2Integration_SingleConsumer_MultipleGasLanes(t *testing.T) {
 	assertNumRandomWords(t, consumerContract, numWords)
 }
 
+func mine(t *testing.T, uni coordinatorV2Universe, db *sqlx.DB) bool {
+	return gomega.NewWithT(t).Eventually(func() bool {
+		uni.backend.Commit()
+		var txs []bulletprooftxmanager.EthTx
+		err := db.Select(&txs, `SELECT * FROM eth_txes WHERE eth_txes.state = 'confirmed' LIMIT 1`)
+		require.NoError(t, err)
+		t.Log("num txs", len(txs))
+		for _, tx := range txs {
+			if tx.Meta != nil {
+				meta := &bulletprooftxmanager.EthTxMeta{}
+				require.NoError(t, json.Unmarshal(*tx.Meta, meta))
+				if meta.SubID != 0 && tx.State == bulletprooftxmanager.EthTxConfirmed {
+					return true
+				}
+			}
+		}
+		return false
```

Comment:
There is a much simpler way to do this, with just one SQL query.

Reply:
cc @makramkd

Reply:
Fixed @samsondav